src/controller/usersController.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const { userError } = require('../errorsHandlers/user');
+const { error400, error404 } = require('../errorsHandlers/general');
+const { usersService } = require('../services/users');
+
+class UsersController {
+  async postUser(req, res) {
+    const { name } = req.body;
+
+    const error = userError(name);
+
+    if (error.errors.length !== 0) {
+      res.status(400);
+      res.json({ error });
+
+      return;
+    }
+
+    const user = await usersService.createUser(name);
+
+    res.statusCode = 201;
+    res.json(user);
+  }
+
+  async getUsers(req, res) {
+    const data = await usersService.getAll();
+
+    res.statusCode = 200;
+    res.json(data);
+  };
+
+  async getUser(req, res) {
+    const { userId } = req.params;
+
+    if (isNaN(+userId)) {
```

Comment:
Why not to use utility for this? `checkUserId` or `isValidNumericField`. In your case better to use method that return error - it will keep everything in one style.